src/frontend/src/components/utils.js

```diff
@@ -0,0 +1,30 @@
+import { Vector3 } from 'three';
+
+export function randomPointOnSphere() {
+    const theta = Math.random() * Math.PI * 2;
+    const phi = Math.random() * Math.PI * 2;
```

Comment:
where would the *2 be cancelled into? If you mean just using `Math.random()` as a vector component it will produce points uniformly distributed inside a cube, not a sphere.